### PR TITLE
Testing for funtionality

### LINKED ISSUE #2: Search Page
- [ ] Search input functionality.
- [ ] Filtering part.

### PR DESCRIPTION
**Pull request by Maral-Kepenekli**
Fix #2

### CODE REVIEW

File: components/search-page-components/RatingFilter.jsx [hunk 1]
```diff
@@ -1,30 +1,27 @@
 "use client";
 import React, { useState } from "react";
-import Box from "@mui/material/Box";
-import Rating from "@mui/material/Rating";
-import StarIcon from "@mui/icons-material/Star";
+import dynamic from "next/dynamic";
 
-function RatingFilter() {
+const Box = dynamic(() => import("@mui/icons-material/Box"));
+const Rating = dynamic(() => import("@mui/icons-material/Rating"));
```
Comment (wasimtayyan) [inline components/search-page-components/RatingFilter.jsx]: Please update the path to "@mui/material/Rating" from '@mui/icons-material/Rating'.

File: components/search-page-components/RatingFilter.jsx [hunk 2]
```diff
@@ -1,30 +1,27 @@
 "use client";
 import React, { useState } from "react";
-import Box from "@mui/material/Box";
-import Rating from "@mui/material/Rating";
-import StarIcon from "@mui/icons-material/Star";
+import dynamic from "next/dynamic";
 
-function RatingFilter() {
+const Box = dynamic(() => import("@mui/icons-material/Box"));
```
Comment (wasimtayyan) [inline components/search-page-components/RatingFilter.jsx]: Please update the path from '@mui/icons-material/Box' to "@mui/material/Box".

**Review by wasimtayyan — COMMENTED** [review]
Could you please update the code that I commented on to resolve the Netlify conflict? Thank you.

**Comment by vuslatsena** [comment]
Good work Maral, nice job. ⭐️
I have only 1 question. When I leave 1 space in the search section, the courses are listed. Is this logic correct?
https://github.com/202303-PRM-TR-FEW/Learning-app-team-5/assets/47564075/5de7aa8f-ed1a-4feb-a576-c6d75cab80e0 (video)

**Pull request closed without merging**